dashboard/web/components/wallet/Wallet2.vue

```diff
@@ -0,0 +1,64 @@
+<template>
```

Comment:
what's the difference between Wallet.vue and Wallet2.vue?

Reply:
different styles. 😂
designed some components before, different styles wallet,  now no need.